src/util/login.js

```diff
@@ -0,0 +1,51 @@
+import * as firebase from 'firebase';
+
+/** Opens login prompt for user and redirects them to the home page if successful. */
```

Comment:
Get rid of the extra asterisk at the beginning of the comment here. That goes for all the rest of the docstrings in this file.